test_config.json

```diff
@@ -7,6 +7,8 @@
 			"test_btp_GAP_CONN_PAIR_3": "Android does not support IO capabilities update"
 		},
 		"skipped_peripheral": {
+			"test_btp_GAP_CONN_DCON_1": "Android has issues with direct connection",
```

Comment:
this was re-added after removed in previous commit, I think you should just undo deletion there